lib/checks/mobile/meta-viewport-scale-evaluate.js

```diff
@@ -14,7 +14,7 @@ function metaViewportScaleEvaluate(node, options) {
 
 		const [key, value] = contentValue.split('=');
 		const curatedKey = key.toLowerCase().trim();
-		let curatedValue = value.toLowerCase().trim();
+		let curatedValue = value && value.toLowerCase().trim();
```

Comment:
This now makes this variable either hold a boolean or a string. This will be confusing when attempting to consume it later on.

Maybe we should do something like this instead:

```suggestion
		let curatedValue = (value || "").toLowerCase().trim();
```

Reply:
Thanks, good point 👍